php/example_code/s3/s3.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * This file is licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+*/
+
+// snippet-sourcedescription:[s3.php demonstrates how to list, create, and delete a bucket in Amazon S3.]
+// snippet-service:[s3]
+// snippet-keyword:[PHP]
+// snippet-keyword:[Amazon S3]
+// snippet-keyword:[Code Sample]
+// snippet-keyword:[listBucketsAsync]
+// snippet-keyword:[createBucketAsync]
+// snippet-keyword:[deleteBucketAsync]
+// snippet-sourcetype:[full-example]
+// snippet-sourcedate:[2017-02-01]
+// snippet-sourceauthor:[AWS]
+// snippet-start:[s3.php.bucket_operations.list_create_delete]
+  require './vendor/autoload.php';
```

Comment:
Is this accurate?

normally it would be `../vendor/autoload.php`

Reply:
It is not installed on Cloud9 by default. As shown on line 28, the assumption here is that it is present in the same directory.